src/conn.ts

```diff
@@ -108,7 +108,25 @@ export class APIConnection implements StatementHandler, Connection {
   }
 
   async exec(query: string, attachments?: Blob[]): Promise<null> {
-    await this.submitStatement(query, attachments);
+    let rs = await this.submitStatement(query, attachments);
+    if (rs.metadata.context != undefined) {
+      let newCtx = rs.metadata.context;
```

Comment:
newCtx here can be null, review to cover both null or undefined ?  >  
https://stackoverflow.com/questions/28975896/is-there-a-way-to-check-for-both-null-and-undefined